controls/radgridview/row-details/how-to/display-window.md

```diff
@@ -74,12 +75,15 @@ As you can see, it inherits from the [INotifyPropertyChanged](http://msdn.micros
 	
 	private Point CalculateWindowLocation(GridViewRow row)
 	{
-	    throw new NotImplementedException();
+            var lastCell = row.Cells[row.Cells.Count - 1];
+            GeneralTransform generalTransform = lastCell.TransformToVisual(Application.Current.RootVisual);
+            return generalTransform.Transform(new Point(lastCell.ActualWidth + 10, 0));
 	}
 {{endregion}}
 
-That’s about all you need to do. You can download a sample project from [here](http://blogs.telerik.com/Libraries/RossenHristov/RowDetailsInRadWindowSources.sflb).
+That's about all you need to do to display the row details in a separate window.
```

Comment:
That's about all you need to do, to display the row details in a separate window. (with a comma)